packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -86,25 +70,71 @@ export const prepareStatusManager = (
     log = makeTracer('Advancer', true),
   } = /** @type {StatusManagerPowers} */ ({}),
 ) => {
-  /** @type {MapStore<PendingTxKey, PendingTx[]>} */
+  /**
+   * Keyed by a tuple of the Noble Forwarding Account and amount.
+   * @type {MapStore<PendingTxKey, PendingTx[]>}
+   */
   const pendingTxs = zone.mapStore('PendingTxs', {
     keyShape: M.string(),
     valueShape: M.arrayOf(PendingTxShape),
   });
 
-  /** @type {SetStore<SeenTxKey>} */
+  /**
+   * Transactions seen *ever* by the contract.
+   *
+   * Note that these are stored in IAVL and grow monotonically. At some point in the future we may want to prune them.
+   *
+   * @type {SetStore<EvmHash>}
+   */
   const seenTxs = zone.setStore('SeenTxs', {
     keyShape: M.string(),
   });
 
+  /**
+   * Transactions that have completed, but are still in vstorage.
+   *
+   * @type {SetStore<EvmHash>}
+   */
+  const storedCompletedTxs = zone.setStore('StoredCompletedTxs', {
+    keyShape: M.string(),
+  });
+
+  /**
+   * @param {EvmHash} txId
+   * @param {CopyRecord} record
```

Comment:
I was expecting a type that includes the enumeration that includes `ADVANCING`.

Reply:
good idea. see `TransactionRecord`